polyfill/lib/calendar.mjs

```diff
@@ -114,7 +114,7 @@ export class Calendar {
 MakeIntrinsicClass(Calendar, 'Temporal.Calendar');
 DefineIntrinsic('Temporal.Calendar.from', Calendar.from);
 
-class ISO8601 extends Calendar {
+class ISO8601Calendar extends Calendar {
```

Comment:
Why this change when it's `Japanese` and not `JapaneseCalendar`?

Reply:
I haven't looked at the Japanese calendar yet.

Reply:
Basically, my question is, do you plan to suffix each class by `Calendar`? It looks weird when it's inconsistent, but if you plan to eventually rename all of them, then it's all good.

Reply:
If we end up going with subclassing, I think suffixing them all makes sense, yes.